docs/docsite/rst/user_guide/playbooks_environment.rst

```diff
@@ -68,6 +68,15 @@ to define an environment hash might be a group_vars file, like so::
       http_proxy: http://proxy.bos.example.com:8080
       https_proxy: http://proxy.bos.example.com:8080
 
+.. versionadded:: 2.10
+
+It's also possible to unset environment variables that are defined by default on the target host by setting a null value for an environment variable::
+
+    - hosts: yourhost
+      environment:
+        http_proxy: ~
+        https_proxy: ~
+
```

Comment:
Include an explanation that this requires support from the shell plugin.

Reply:
Done